lms/views/lti/deep_linking.py

```diff
@@ -76,6 +76,7 @@ def deep_linking_launch(context, request):
             "lti_message_type": "ContentItemSelection",
             "lti_version": request.parsed_params["lti_version"],
         },
+        prompt_for_title=request.product.plugin.misc.deep_linking_prompt_for_title,
```

Comment:
Note that we don't use this while editing.

It's not possible to change the title on the LMS on a launch, we can/should only do that while responding to a deep linking request.